extensions/barrage/src/main/java/io/deephaven/extensions/barrage/util/ArrowToTableConverter.java

```diff
@@ -192,7 +198,7 @@ protected BarrageMessage createBarrageMessage(BarrageProtoUtil.MessageInfo mi, i
         return msg;
     }
 
-    private BarrageProtoUtil.MessageInfo getMessageInfo(byte[] ipcMessage) {
+    private BarrageProtoUtil.MessageInfo getMessageInfo(ByteBuffer ipcMessage) {
```

Comment:
Flatbuffer almost certainly retains references to ByteBuffer. We need to be very careful that nothing retains this flatbuffer object. I think we need to add a big warning in `io.deephaven.extensions.barrage.util.ArrowToTableConverter#createBarrageMessage` that nothing can retain reference to `BarrageProtoUtil.MessageInfo mi`